docs/source/dev_guide.rst

```diff
@@ -89,6 +94,10 @@ You can also develop on any Posix compatible system (Linux / MacOS) in roughly t
 same way you can on a Raspberry Pi.  The emulated hardware and networking features 
 will work differently so this is mostly useful for UI/Catalog feature development.
 
+Note that you can develop on Windows by activating Windows Subsystem for Linux (WSL2) 
+and installing Ubuntu from the Microsoft Store. The window launched by PiFinder will 
+be fully integrated into your windows desktop. 
```

Comment:
This is a great hint, I have little windows experience so I am very appreciative of any hints/tricks that can be offered to widen the scope of potential contributers 🙏